src/utils/deploy.ts

```diff
@@ -34,6 +40,8 @@ export function validateTests(testLevel: TestLevel, tests: Nullable<string[]>):
   return true;
 }
 
+type CachedOptions = Omit<Options, 'wait' | 'target-org'> & { 'target-org': string; wait: number } & JsonMap;
```

Comment:
I understand what you did with changing the type/requiredness for wait, but am not following what you're doing with target-org

Reply:
That's a vestige of an older implementation - I'll remove it